landingzones/tests/test_permissions_taskflow.py

```diff
@@ -0,0 +1,388 @@
+"""Tests for UI view permissions with taskflow"""
+
+from django.test import override_settings
+from django.urls import reverse
+
+# Projectroles dependency
+from projectroles.models import SODAR_CONSTANTS
+from projectroles.tests.test_permissions import (
+    TestPermissionMixin,
+)
+
+from landingzones.tests.test_views_taskflow import LandingZoneTaskflowMixin
```

Comment:
Make sure the imports are in the expected order:

- projectroles
- samplesheets
- taskflowbackend
- landingzones